test/network.schema

```diff
@@ -5,7 +5,10 @@
         "devnet": {
             "type": "boolean"
         },
-        "alpha": {
+        "alphaGoerli": {
```

Comment:
Can we remove `alphaGoerli` from the list of test network options? So that we are sure we don't use it somewhere by mistake.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/258/commits/d10febff448a7a635795f576e6ecbd81af36bf6e